### PR TITLE
Rely only on the model for resolving device type aliases

### PR DESCRIPTION
**Pull request by thgreasi**
Change-type: minor
Signed-off-by: Thodoris Greasidis <thodoris@balena.io>

### CODE REVIEW

File: src/features/device-types/device-types.ts [hunk 1]
```diff
@@ -1,10 +1,9 @@
 import * as arraySort from 'array-sort';
 import * as _ from 'lodash';
 
-import * as deviceTypesLib from '@resin.io/device-types';
+import type { DeviceType as DeviceTypeJson } from '@resin.io/device-types';
```
Comment (thgreasi) [inline src/features/device-types/device-types.ts:4]: @dfunckt btw after this PR, we no longer use the code in `@resin.io/device-types` and given that the types that we import are already in custom typings in oB, we can just drop it.

Comment (dfunckt) [reply src/features/device-types/device-types.ts:4]: Sounds great, why don't we just drop it with this PR?

Comment (thgreasi) [reply src/features/device-types/device-types.ts:4]: Will do :+1:

**Comment by thgreasi** [comment]
@balena-ci I self-certify!

**Pull request merged**